kolibri/core/auth/serializers.py

```diff
@@ -133,13 +133,23 @@ class Meta:
 
 class PublicFacilitySerializer(serializers.ModelSerializer):
     learner_can_login_with_no_password = serializers.SerializerMethodField()
+    learner_can_sign_up = serializers.SerializerMethodField()
 
     def get_learner_can_login_with_no_password(self, instance):
         return instance.dataset.learner_can_login_with_no_password
 
+    def get_learner_can_sign_up(self, instance):
+        return instance.dataset.learner_can_sign_up
```

Comment:
I know it was like that when you got here - but these kind of n queries chained look ups always bother me.